### DISCUSSION TITLE
Can PRs successfully run GHA actions?

### DISCUSSION THREAD

**Discussion by gsong** [ip-tools/python-epo-ops-client#82] [Q&A]
@amotl See <https://github.com/ip-tools/python-epo-ops-client/pull/81> and <https://github.com/ip-tools/python-epo-ops-client/actions/runs/7671541277/job/20910120163?pr=81>, is there any way for external contributor PRs to have successful CI?

**Comment by amotl** [comment]
Hi @gsong. I can't find what your issue was about. If CI/GHA uses secrets, which ours does, and context isolation applies, it is not easy that PRs by external contributors have access to them, for security reasons.

- https://stackoverflow.com/questions/71423745/how-can-i-make-github-actions-work-with-prs-made-by-external-contributors
- https://github.com/orgs/community/discussions/62316
- https://dvc.org/blog/testing-external-contributions-using-github-actions-secrets
